.github/workflows/integration-build-product.yml

```
@@ -674,6 +692,24 @@ jobs:
         env:
           SIMULATION_ID: "CI"
 
+      - name: Collect Install logs
+        continue-on-error: true
+        run: |
+          # cleanup
+          rm -rf install_logs
+
+          mkdir -p install_logs
+          journalctl -u movai-service -t mobros -n 100 > install_logs/spawner-firmware.log
+          journalctl -u movai-service > install_logs/movai-service.log
+
+      - name: Stash Install logs artifacts
+        continue-on-error: true
+        if: always()
+        uses: actions/upload-artifact@v3
+        with:
+          name: install_logs
```

It should be different no? Not sure what happens if they are the same
```suggestion
          name: install_simulator_logs
```